tests/functions/test_visualizations.py

```diff
@@ -227,8 +227,8 @@ def test_cytof_clustergrammer_end_to_end(monkeypatch, metadata_df, upload_type):
 
     # Based on the construction of metadata_df...
     assert col_cats == [
-        ("Participant ID: CTTTTPP", "Cohort: Arm_A", "Collection Event: Event1"),
-        ("Participant ID: CTTTTPP", "Cohort: Arm_A", "Collection Event: Event2"),
+        ("CIMAC Participant ID: CTTTTPP", "Cohort: Arm_A", "Collection Event: Event1"),
+        ("CIMAC Participant ID: CTTTTPP", "Cohort: Arm_A", "Collection Event: Event2"),
     ]
```

Comment:
could you add to the test those extra categories to see how they will look like here? 

I'm asking because 
1. I wonder if we should also drop "json-path" prefixes - i.e. right now it looks like "arbitrary_trial_specific_clinical_annotations.Treatment" 
2. What if there are collisions in column names (after all the `.replace(_)` etc) - i.e. "arbitrary_trial_specific_clinical_annotations.Cohort" (with values `Colon|Lung`) vs "Cohort" (`Arm A|B`)

Reply:
^agreed - it'd be great to see tests for every transformation that `_metadata_to_categories` might make

Reply:
^ also - it doesn't show test coverage, we don't have that here?

Reply:
added tests